suite-native/module-accounts/src/screens/AccountDetailScreen.tsx

```diff
@@ -34,33 +34,37 @@ export const AccountDetailScreen = ({
     const accountTransactions = useSelector((state: TransactionsRootState) =>
         selectAccountTransactions(state, accountKey),
     );
+    const [page, setPage] = useState(1);
     const dispatch = useDispatch();
 
-    useEffect(() => {
+    const fetchMoreTransactions = () => {
         if (!account) return;
-        const promise = dispatch(
+        dispatch(
             fetchTransactionsThunk({
                 account,
-                page: 1,
-                perPage: 100, // TODO add support for pagination.
+                page,
+                perPage: TX_PER_PAGE,
             }),
         );
-        return () => {
-            promise.abort();
-        };
-    }, [account, dispatch]);
+        setPage(page + 1);
+    };
+
+    useMount(() => {
+        fetchMoreTransactions();
+    });
+
+    const paginatedTransactions = accountTransactions.slice(1, TX_PER_PAGE * page);
```

Comment:
We don't need this we call always show everything what we have, because SectionList takes care of optimalizations.

Reply:
[f3a77d8](https://github.com/trezor/trezor-suite/pull/6319/commits/f3a77d8c2b615620ef6ce5d2c88de78da97ac147)